src/Serilog/Configuration/LoggerDestructuringConfiguration.cs

```diff
@@ -203,4 +206,26 @@ public LoggerConfiguration ToMaximumCollectionCount(int maximumCollectionCount)
         _setMaximumCollectionCount(maximumCollectionCount);
         return _loggerConfiguration;
     }
+
+    /// <summary>
+    /// If no explicit destructuring hint was given for the property, use the given
+    /// destructuring as fallback.
+    /// </summary>
+    /// <param name="destructuringFallback">The fallback destructuring.</param>
+    /// <param name="type">Type to define the fallback for.</typeparam>
+    /// <returns>Configuration object allowing method chaining.</returns>
+    public LoggerConfiguration WhenNoOperator(Type type, DestructuringFallback destructuringFallback)
```

Comment:
It would be possible to have these use `Destructuring` and `bool` directly in the user-facing API, but I just used `DestructuringFallback` as forward-compatible version.